core/services/validation.py

```
@@ -80,13 +83,15 @@ class ForeignNumberCheck(ValidationCheck):
     """
     name = "foreign_number_check"
     
-    def _execute(self, mobile_number: str, allowed_countries: list,
-                 **kwargs) -> Tuple[int, Optional[str]]:
+    def _execute(self, **kwargs: Any) -> Tuple[int, Optional[str]]:
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)